webapp/packages/plugin-navigation-tree/src/NavigationTree/NavigationNodeControl.m.css

```diff
@@ -0,0 +1,41 @@
+.treeNodeControl.dragging {
+  opacity: 0.6;
+}
```

Comment:
duplicate (working one) of webapp/packages/plugin-navigation-tree/src/NavigationTree/ProjectsRenderer/NavigationNodeProjectControl.m.css

Reply:
please remove it here